Source/Extensions/Blazorise.DataGrid/DataGrid.razor.cs

```diff
@@ -127,6 +138,12 @@ internal void Hook( DataGridAggregate<TItem> aggregate )
 
         protected override Task OnAfterRenderAsync( bool firstRender )
         {
+            if ( !MultiSelect && SelectedRows is object )

```

Comment:
Also checking `SelectedRows is object`.

Reply:
Same thing as before. It's a null check. 
It's kind of a personal preference. Would you like that I change it to != null since that's your standard way of null checking.